js/cfi_generator.js

```diff
@@ -193,10 +193,12 @@ var obj = {
         this.validatePackageDocument(packageDocument, contentDocumentName);
 
         // Get the start node (itemref element) that references the content document
-        $itemRefStartNode = $("itemref[idref='" + contentDocumentName + "']", $(packageDocument));
+        var $itemRefStartNode = $($(packageDocument.getElementsByTagNameNS('*', 'itemref')).toArray().find(function (node) {
+            return node.getAttribute('idref') === contentDocumentName;
+        }));
```

Comment:
Support for `array.find()` in web browsers is not broad enough, it seems:
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
IE does not have support at all. can we use underscore or jQuery find() instead?